extra_tests/snippets/builtin_str.py

```
@@ -220,6 +222,14 @@
 assert "abc\t12345\txyz".expandtabs() == "abc     12345   xyz"
 assert "-".join(["1", "2", "3"]) == "1-2-3"
 assert "HALLO".isupper()
+assert not "123".isupper()
+assert not "123".islower()
+assert not "\U0001f431".isupper();
```

Flake8 (7.3.0)

[error] statement ends with a semicolon

(E703)